code/game/objects/items/tools/soldering_tool.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/tool/solderingtool
+	name = "soldering tool"
+	desc = "A hand tool to fix combat robot's trauma. You do not need welding goggles for this and you need medical skills for this."
+	icon = 'icons/obj/items/surgery_tools.dmi'
+	icon_state = "alien_hemostat"
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/tool/solderingtool/attack(mob/living/carbon/human/H, mob/user)
+	if(!istype(H)||user.a_intent != INTENT_HELP)
+		return ..()
+
+	var/datum/limb/affecting = user.client.prefs.toggles_gameplay & RADIAL_MEDICAL ? radial_medical(H, user) : H.get_limb(user.zone_selected)
+	if(!affecting)
+		return TRUE
+
+	if(affecting.limb_status != LIMB_ROBOT)
+		balloon_alert(user, "Limb not robotic")
+		return TRUE
+
+	if(!affecting.burn_dam && !affecting.brute_dam)
+		balloon_alert(user, "Nothing to fix!")
+		return TRUE
+
+	if(user.do_actions)
+		balloon_alert(user, "Already busy!")
+		return TRUE
+
+	var/repair_time = 1.5 SECONDS
+	if(H == user)
+		repair_time *= 1.5
+
+	if(user.skills.getRating("engineer") < SKILL_ENGINEER_ENGI)
+		user.visible_message(span_notice("[user] fumbles around figuring out how to solder the wounds on [src]."),
+		span_notice("You fumble around figuring out how to solder the wounds on [src]."))
+		var/unskilled_delay = 200 SECONDS * ( SKILL_ENGINEER_ENGI - user.skills.getRating("engineer") )
```

Comment:
You sure this should be `*` and not `/`
Default marine engi skill is 0, `SKILL_ENGINEER_ENGI` is 3, so for a default marine it would take 600 seconds.
TEN FUCKING MINUTES TO USE THIS TOOL ONCE IF YOU ARE A STANDARD MARINE! YOU WOULD HAVE TO STAND STILL TEN MINUTES!
Corpman has the same delay because they aint got engi skills either.

Reply:
resolved, baby